presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java

```diff
@@ -290,7 +292,8 @@ private LocalQueryRunner(Session defaultSession, FeaturesConfig featuresConfig,
         this.accessControl = new TestingAccessControlManager(transactionManager);
         this.pageSourceManager = new PageSourceManager();
 
-        this.expressionCompiler = new ExpressionCompiler(metadata);
+        this.pageFunctionCompiler = new PageFunctionCompiler(metadata, false);
```

Comment:
The default is true so it is tested by all of the integration tests.  I set the rest of the tests to false to test the disabled case.